.azure-pipelines/templates/installer-tests.yml

```
@@ -31,6 +31,13 @@ jobs:
     pool:
       vmImage: $(imageName)
     steps:
+      - powershell: |
+          $currentVersion = (Get-Host | Select-Object Version).Version
```

It may not matter, but I found a couple websites recommending people use `$PSVersionTable.PSVersion` instead of `Get-Host`. See [this](https://adamtheautomator.com/check-powershell-version/) or [this](https://stackoverflow.com/questions/1825585/determine-installed-powershell-version).

What do you think? It seems like making the following change to this line should work just fine and should avoid any potential differences discussed in those posts:
```suggestion
          $currentVersion = $PSVersionTable.PSVersion
```

---

I hesitated with this approach. Some sites are recommending the `Get-Host` call, others `$PSVersionTable`. Honestly I could not decide between both, it seems strictly identical, and I am pretty sure that Powershell is using `Get-Host` internally to feed `$PSVersionTable`.

Since `$PSVersionTable` gives a little less indirections, let's use it.